docs/en/workshop/a-molecule-unfolding/batch-evaluation.md

```diff
@@ -0,0 +1,220 @@
+## Run Batch Evaluation
+
+We will run batch evaluation through AWS Step Functions workflow and view the result via AWS QuickSight dashboard
+
+### Get Step Functions link from deployment output
+
+![deployment output](../../images/deploy-output-stepfunc.png)
+
+Click the Step Functions link, you will be navigated to AWS Step Functions console.
+
+### Start Execution
+
+In your AWS Step Functions console, click **Start execution** button, the screen is shown as below:
+
+![ start execution step function ](../../images/batch-start-execution.png)
+
+1. (optional) Input evaluation input
+
+     - It will use default input if you do not input anything.
+     - If you want to customize the batch evaluation, please refer to the [Input specification](#input-specification) in this section.
+
+1. Click **Start execution**, start batch evaluation
+    
+    The screen is shown as below:
+    ![ execution step function ](../../images/batch-execution.png)
+
+1. Wait for complete
```

Comment:
did you render the docs locally? are you sure the number bullet work as expected?

Reply:
yes.